package.json

```diff
@@ -78,10 +78,10 @@
     "shelljs": "^0.7.0",
     "style-loader": "^0.13.0",
     "ts-helpers": "^1.1.1",
-    "tslint": "^3.4.0",
-    "tslint-loader": "^2.1.0",
+    "tslint": "^4.3.1",
+    "tslint-loader": "^3.3.0",
     "typedoc": "^0.5.1",
-    "typescript": "2.0.10",
+    "typescript": "2.1.5",
```

Comment:
revert this one please. TS doesn't work with AOT (yes, I don't have it here yet) but I know people adding AOT on this and they will be confused.